moped-editor/src/views/projects/projectView/ProjectPurchaseOrderTable.js

```diff
@@ -0,0 +1,256 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useQuery, useMutation } from "@apollo/client";
+
+// Material
+import {
+  Button,
+  CircularProgress,
+  Snackbar,
+  Typography,
+} from "@material-ui/core";
+import { Alert } from "@material-ui/lab";
+import {
+  AddCircle as AddCircleIcon,
+  DeleteOutline as DeleteOutlineIcon,
+  EditOutlined as EditOutlinedIcon,
+} from "@material-ui/icons";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import MaterialTable, {
+  MTableEditRow,
+  MTableAction,
+} from "@material-table/core";
+import typography from "../../../theme/typography";
+
+import { PAGING_DEFAULT_COUNT } from "../../../constants/tables";
+
+// Error Handler
+import ApolloErrorHandler from "../../../components/ApolloErrorHandler";
+
+import {
+  PURCHASE_ORDER_QUERY,
+  UPDATE_PURCHASE_ORDER,
+  ADD_PURCHASE_ORDER,
+  DELETE_PURCHASE_ORDER,
+} from "../../../queries/funding";
+
+const useStyles = makeStyles(theme => ({
+  addRecordButton: {
+    position: "absolute",
+    top: "1rem",
+    right: "1rem",
+  },
+}));
+
+const ProjectPurchaseOrderTable = () => {
+  // addAction Ref - mutable ref object used to access add action button imperatively.
+  const addActionRef = React.useRef();
```

Comment:
I think this ref can go since it isn't used to access the add button anywhere. I could be missing something though